api/client/gro_client.py

```diff
@@ -212,7 +212,7 @@ def convert_unit(self, point, target_unit_id):
             new unit_id. Other properties are unchanged.
 
         """
-        if point.get('unit_id') is None or point.get('unit_id') == target_unit_id:
+        if point['value'] is None or point.get('unit_id') is None or point.get('unit_id') == target_unit_id:
```

Comment:
As noted in shippable https://app.shippable.com/github/gro-intelligence/api-client/runs/714/1/tests this is failing on `assert client.convert_unit({}, 36) == {}`

This happens when there's no data. Instead of `[]` like you might expect, you get `[{}]` so you can't assume there's a 'value' property in that dict. Have to use `point.get('value') is None` instead

Reply:
Gotcha, will change